Src/AutoFixture.xUnit.net2.UnitTest/TypeWithCustomizationAttributes.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Ploeh.TestTypeFoundation;
+
+namespace Ploeh.AutoFixture.Xunit2.UnitTest
+{
+    internal class TypeWithCustomizationAttributes
+    {
+        public void CreateWithFrozenAndGreedy([Frozen, Greedy] ConcreteType sut)
+        {
+        }
+
+        public void CreateWithGreedyAndFrozen([Greedy, Frozen] ConcreteType sut)
+        {
+        }
```

Comment:
Make sense to me. Added https://github.com/AutoFixture/AutoFixture/pull/590/commits/1e62a5ea36f289a9c3399236d8dab83b1f833ce9.